src/components/parts/home/LatestPosts.astro

```diff
@@ -1,11 +1,12 @@
 ---
 import Button from "@atoms/Button.astro";
 import PostEmbed from "@atoms/PostEmbed.astro";
-
+import type { MarkdownInstance } from "astro";
 import i18next from "i18next";
 import { localizePath, sortByDescending } from "src/util/Util";
+import type { BlogFrontMatter } from "@layouts/Post.astro"
 
-const allPosts = (await Astro.glob("/src/pages/**/blog/*.md")).filter((p) =>
+const allPosts = (Object.values(import.meta.glob("/src/pages/**/blog/*.md", {eager: true})) as [MarkdownInstance<BlogFrontMatter>]).filter((p) =>
```

Comment:
Is `eager: true` relevant for markdown files ?

Reply:
I think so. All of the examples in the docs (like [this one](https://docs.astro.build/en/guides/imports/#importmetaglob) and [this one](https://docs.astro.build/en/guides/upgrade-to/v5/#what-should-i-do-3) use Markdown files and `{eager: true}`. I think it just loads the imported data immediately rather than having to do it later.